project/huroy5518/README.md

```diff
@@ -0,0 +1,69 @@
+# Graph Partitioning for Solving PDEs
+A library that can perform graph partitioning for solving PDE purpose. 
+
+## Basic Information
+For solving PDEs numerically, there are some famous methods to achieve the goal. FEM(Finite element method) is one of them.
+
+Graph partitioning can be used in FEM to divide the domain into subdomain or elements. Which can then be sloved independently or in parallel. So a good design of graph partition algorithm can greatly the performance for solving PDE numerically in some cases.
```

Comment:
It's not entirely correct to say graph partitioning solves a PDE.  It's actually used to divide the spatial domain into smaller pieces for parallel processing.

It does not invalidate your plan for coding.  Just don't mistake the problem you try to solve.